.gitignore

```diff
@@ -2,7 +2,7 @@
 .kotlin/
 .idea/*
 !.idea/icon.svg
-.intelliJPlatform/
+.intellijPlatform/
```

Comment:
Are you sure about that?

Reply:
That was at least the folder my IDE generated

Reply:
Oh yeah my bad that's what I have too
I guess the gitignore is case-insensitive then